cli/src/installers/envVars.ts

```diff
@@ -38,7 +38,7 @@ DATABASE_URL=file:./db.sqlite
 # Next Auth
 # You can generate the secret via 'openssl rand -base64 32' on Linux
 # More info: https://next-auth.js.org/configuration/options#secret
-NEXTAUTH_SECRET=supersecret
+# NEXTAUTH_SECRET=
```

Comment:
`z.string()` actually passes with an empty string, so im not sure its necessary to comment it out here?